### PR TITLE
166 display number of difference in pop up windows for pluto graphs

### PR DESCRIPTION
**Pull request by Oysters1874**
I modified the tooltip content to display difference instead of percent change. But as for the y-axis, we are still displaying percent change. However I realized one thing that before we are using the previous version - current version to calculate the percent change, which is not appropriate for this calculation.

### CODE REVIEW

File: src/pluto/pluto.py [hunk 1]
```diff
@@ -477,15 +477,16 @@ def generate_graph(v1, v2):
...
+                diff=[v2[i] - v1[i]  for i in y]
                 real_v1 = [v1[i] for i in y]
                 real_v2 = [v2[i] for i in y]
                 hovertemplate = "<b>%{x}</b> %{text}"
                 text = []
                 for n in range(len(y)):
                     text.append(
-                        "Percent Change: {:.2f}%<br>Prev: {} Current: {}".format(
-                            x[n],
+                        "Diff: {} Prev: {} Current: {}".format(
```
Comment (mbh329) [inline src/pluto/pluto.py:488]: One thing I just thought of is that I think it reads better if `Current` is first followed by `Prev`

Comment (Oysters1874) [reply src/pluto/pluto.py:488]: oh right, that will look better.

**Comment by mbh329** [comment]
I think this graph is actually more messed up than I thought

**Comment by Oysters1874** [comment]
> I think this graph is actually more messed up than I thought - checking the numbers in the table the current version for factory area should display the 116.87m value but right now we have this backwards

realized one thing, before we have reversed the place for the current and previous version in the tooltip, so I thought v1 is the previous. I have modified it to the right sequence.

**Comment by Oysters1874** [comment]
> > I think this graph is actually more messed up than I thought - checking the numbers in the table the current version for factory area should display the 116.87m value but right now we have this backwards
> 
> realized one thing, before we reversed the place of current and previous version in the tooltip. I have modified it to the right sequence.

**Comment by mbh329** [comment]
Great, thanks for fixing that! I think this is almost done. I actually don't mind the y axis showing the pct change while the hover text shows the difference, I think it adds a clear extra dimension to the graph but curious what other people think

**Comment by SashaWeinstein** [comment]
I think the pct change on the y axis is the way to get all the lines on the same plot when some columns are on the scale of thousands and other billions right? I think it's fairly readable

**Comment by Oysters1874** [comment]
> I think the pct change on the y axis is the way to get all the lines on the same plot when some columns are on the scale of thousands and other billions right? I think it's fairly readable

Yea, if we use the difference as the y-axis, most of the lines may completely overlap with other each. That will be hard to read.

**Review by SashaWeinstein — APPROVED** [review]
Looks good to me! I have one change I think might be cool, will open up a PR to get feedback

**Pull request merged**